### ISSUE TITLE
Community Chart Suggestion: Nextcloud Talk Recording

### ISSUE THREAD

**Issue by pabloeisenhut** [truenas/apps#1194]

**Comment by stavros-k** [comment]
Are there any documentation on how is this being setup? Assuming the hpb is available after #1193 is completed.

**Comment by stavros-k** [comment]
> There are [installation instructions](https://github.com/nextcloud/nextcloud-talk-recording/blob/main/docs%2Finstallation.md) and a [pre-made Docker Compose](https://github.com/nextcloud/nextcloud-talk-recording/blob/main/docker-compose%2Fdocker-compose.yml)... But otherwise you can also take a look at the [Nextcloud AIO](https://github.com/nextcloud/all-in-one/blob/main/manual-install%2Flatest.yml) repository where the official Docker installation paths are used, only the images are separately packed and included, but you can also view these Dockerfiles.
> 
> Otherwise, if you want to save yourself the work, you can also use the packed Nextcloud Talk Recording Image from the manual Nextcloud AIO installation.

Installation instructions does not seem to be targeting docker, and the premade compose file, uses a local dockerfile, not a published image.
AIO seems to have separate containers, but will still need to see how it all interconnects/plumbs with Nextcloud Talk and other components. ie how do you configure nextcloud to use it? What needs to be client exposed vs inter-container only, etc.
If someone could compile a list of sources would be nice, otherwise I will have to dedicate time for that, which might delay it a bit.

**Comment by FlattusBlastus** [comment]
https://github.com/nextcloud/nextcloud-talk-recording/blob/main/docs/installation.md#manual-installation
The recording server has the following non-Python dependencies:

FFmpeg
Firefox*
[geckodriver](https://github.com/mozilla/geckodriver/releases) (on a [version compatible with the Firefox version](https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html))
PulseAudio
Xvfb

You'd have to go with an entirely separate app in TrueNAS to provide a functional setup. Adding an entire browser to the NC App seems like a no-go. Recommend close request with Won't fix - Community will need to provide solution.

**Comment by stavros-k** [comment]
I'm not sure why you link bare metal / native installation docs. This is not how apps are installed.
